odl/space/pspace.py

```diff
@@ -1229,7 +1238,7 @@ def inner(self, x1, x2):
             dtype=x1[0].space.dtype, count=len(x1))
 
         inner = self.const * np.sum(inners)
-        return x1.space.field.element(inner)
+        return x1.space.field.element(inner.real)
```

Comment:
Are you sure about this? What about complex spaces?

Reply:
How did that slip in??